### DISCUSSION TITLE
Add support for Bookmarks in User Directory

### DISCUSSION THREAD

**Discussion by gloribus** [directus/directus#24947] [Feature Requests]
### Describe the Bug

When I select the Directus Users collection in the bookmarks (/admin/settings/presets/+), this bookmark does not display anywhere

### To Reproduce

1) Go /admin/settings/presets/+
2) Add bookmark with Directus Users collection

### Hosting Strategy

Self-Hosted (Custom)

**Comment by paescuj** [comment]
The preset for Directus Users configures the layout for `/admin/users`.

**Comment by gloribus** [comment]
> The preset for Directus Users configures the layout for `/admin/users`.

But this does not change layout for /admin/users. I tried adding a global filter but it doesn't show up

**Comment by iksent** [comment]
I also experience the same problem: I would like to add custom view for users, so I create a new Bookmark with Custom template and filter for Directus Users, and it's not displayed anywhere.

**Comment by paescuj** [comment]
> But this does not change layout for /admin/users. I tried adding a global filter but it doesn't show up

Classic misunderstanding 😄 Reopening again 👍

**Comment by br41nslug** [comment]
I think currently bookmarks are only applied in the "Content Module" which does not display core collections.
